libtiledbvcf/src/read/tsv_exporter.cc

```diff
@@ -210,8 +210,8 @@ bool TSVExporter::export_record(
             }
             std::string str(s[0], val.ndst);
             os << str;
-            free(s[0]);
-            free(s);
+            hts_free(s[0]);
```

Comment:
Is there a reason to use `hts_free` vs `ks_free` here?

Reply:
ks_free is not currently available outside the library (static inline in header)
while hts_free() is already exported and available to clients.
(hts_free() is also a direct replacement for free(), while ks_free() would require argument changes.)

Reply:
SGTM as long as VCF maintainers are ok with it, thanks.